pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -635,3 +651,31 @@ func notFound(err error) bool {
 	}
 	return false
 }
+
+// generateValuesYaml generates a YAML string containing any
+// necessary values to override defaults in values.yaml. If
+// no defaults need to be overwritten, an empty string will
+// be returned.
+func generateValuesYaml() (string, error) {
+	httpProxy := os.Getenv("HTTP_PROXY")
+	httpsProxy := os.Getenv("HTTPS_PROXY")
+	noProxy := os.Getenv("NO_PROXY")
+
+	if httpProxy == "" && httpsProxy == "" && noProxy == "" {
+		// no proxy related environment variables set
+		return "", nil
+	}
```

Comment:
nit: is there any reason why we need to specifically pass in `""` here after checking the values instead of just letting it pass in:
```
{
 	"httpProxy": "",
 	"httpsProxy": "",
 	"noProxy": ""
 }
```

Reply:
If there's no specific reason for this, I think it would be cleaner to just pass in the `os.Getenv` calls directly while creating the struct.

Reply:
I just didn't see the point in marshaling and passing a values yaml to the cli for no reason. I think your suggestion would be clean as well but view these approaches about equal: either we pass a redundant values yaml to cli or we have a few extra lines.

Reply:
Went ahead and changed this, since if more env variables are eventually added it may be unreasonable to check if they are all empty anyway.